src/js/providers/video-listener-mixin.js

```diff
@@ -105,7 +108,6 @@ const VideoListenerMixin = {
     },
 
     playing() {
-        this.setState(STATE_PLAYING);
```

Comment:
This makes sense for iOS, but not for Chrome and Firefox as the behavior you are observing is iOS/Safari specific behavior.

Reply:
makes sense, I could prevent the state change if seeking === true, would you be ok with that ? ( context:  this was removed because a playing event is fired in between seeking and seeked, causing the player to exit the buffering state).
so `playing() {
         if (seeking) {
                return;
         }
         this.setState(STATE_PLAYING);
}`

Reply:
Yes. That makes sense.